lib/services/user.dart

```diff
@@ -1,17 +1,20 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 
 class DatabaseService {
-
   final String uid;
   DatabaseService({this.uid});
 
   //collection reference
-  final CollectionReference userCollection = Firestore.instance.collection('users');
+  final CollectionReference userCollection =
+      Firestore.instance.collection('users');
 
-  Future updateUserData(String name, int age) async {
+  Future updateUserData(
+      String firstName, String lastName, String username, int bp) async {
     return await userCollection.document(uid).setData({
-      'name': name,
-      'age':age,
+      'firstName': firstName,
+      'lastName': lastName,
+      'username': username,
+      'bp': bp
```

Comment:
What's bp?

Reply:
bp = ball points
I think this is the term that Raymond used when he was talking about earning points through playing games

Reply:
gotcha, let's rename that so it's clear, always avoid abbreviations for variable names if it's not too long